<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditWidget</class>
 <widget class="QWidget" name="ItemEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="AdvLineEdit" name="edName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Desc:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="AdvLineEdit" name="edDesc"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Article:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="AdvLineEdit" name="edArticle"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Unit of meas.:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="AdvComboBox" name="cmbUnit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="AdvComboBox" name="cmbType"/>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ckActive">
       <property name="text">
        <string>Is Active</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/advlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>AdvComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/advcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
